develop-docs/sdk/telemetry/feedback.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: Feedbacks
+sidebar_order: 4
+---
+
+This document is meant for Sentry SDK developers and maintainers of the Feedback ingestion pipeline.
+
+## Feedback Protocol
+
+Item type `"feedback"`. This Item contains an [event payload](/sdk/data-model/event-payloads/) encoded in JSON, with an additional `feedback` context object.
+
+## `"feedback"` Item
+
+### Feedback Context Attributes
+
+| Key                    | Type                      | Description                                                                                                                                                                                          |
+| ---------------------- | ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| message                | string                    | Required. Comments of the user, describing what happened and/or sharing feedback. The max length is **4096 characters**.                                                                             |
+| contact_email          | string                    | The email of the user who submitted the feedback. If excluded, Sentry attempts to fill this in with user context. Anonymous feedbacks (no name or email) are still accepted.                         |
```

Comment:
`Sentry attempts to fill this in with user context.`
Does this mean Sentry SDK? Or is it relay doing it?
Same for name